src/mon/MgrMonitor.cc

```diff
@@ -60,10 +66,19 @@ void MgrMonitor::update_from_paxos(bool *need_bootstrap)
     }
 
     check_subs();
-  }
 
-  // feed our pet MgrClient
-  mon->mgr_client.ms_dispatch(new MMgrMap(map));
+    if (map.get_available() && !old_available) {
```

Comment:
I think this could miss an update if, say,
- mon.b is quorum, active mgr.x
- mon.b drops out of quorum
- other mons transition to mgr.y
- mon.b rejoins quorum